kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/onboarding-form.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+
+  <div>
+    <h1 class="onboarding-form-header">
+      {{ header }}
+    </h1>
+
+    <form @submit.prevent="$emit('submit')">
+      <fieldset class="onboarding-form-fields">
+        <legend class="onboarding-form-legend">
+          <span v-if="hasDescription" class="onboardng-form-description">
+            <slot name="description"> {{ description }} </slot>
+          </span>
+          <span v-else class="visuallyhidden"> {{ header }} </span>
+        </legend>
+
+        <slot />
+      </fieldset>
+
+      <k-button
```

Comment:
Disable while form is submitting.

Reply:
On submit, the entire screen actually changes to a loading screen. Will do, though